provider/resources.go

```diff
@@ -400,6 +400,9 @@ func preConfigureCallbackWithLogger(credentialsValidationRun *atomic.Bool, gcpCl
 			"CLOUDSDK_CORE_PROJECT",
 		})
 		if project == "" {
+			if project == "No Project" {
```

Comment:
I'm a fan of using a descriptive string here!

non-blocking: Despite the fact that I was the one who suggested `No Project` I'm thinking now that may be confusing, because folks will still need a Project; it's just configured elsewhere.

How about `DisableGlobalProjectWarning` or `NoGlobalProject`? That's a bit closer to the intent maybe.

Reply:
I like `DisableGlobalProjectWarning` I will swap it out to that. I think its much clearer. I also didn't like the space in the config value.